packages/date/src/DateRange.js

```diff
@@ -68,18 +70,27 @@ const DateRange = ({
   customArrowIcon,
   ...attributes
 }) => {
-  const { setFieldValue, setFieldTouched } = useFormikContext();
+  const { setFieldValue, setFieldTouched, validateField } = useFormikContext();
   const [{ value = {} }, metadata] = useField({ name, validate });
   const [focusedInput, setFocusedInput] = useState(null);
 
   const calendarIconRef = useRef();
 
   const startId = `${(id || name).replace(/[^a-zA-Z0-9]/gi, '')}-start`;
-
   const endId = `${(id || name).replace(/[^a-zA-Z0-9]/gi, '')}-end`;
 
-  const startValue = get(value, startKey);
-  const endValue = get(value, endKey);
+  const startValue = value[startKey] || '';
```

Comment:
i believe test is broken because of this change. s/b
```
const startValue = value[startKey] || undefined;
const endValue = value[endKey] || undefined;
```